selfdrive/car/toyota/values.py

```diff
@@ -848,8 +848,10 @@ class CAR:
       b'\x01F15264872300\x00\x00\x00\x00',
       b'\x01F15264872400\x00\x00\x00\x00',
       b'\x01F15264872500\x00\x00\x00\x00',
+      b'\x01F152648C6300\x00\x00\x00\x00',
     ],
     (Ecu.engine, 0x700, None): [
+      b'\x01896630EA1000\x00\x00\x00\x00897CF4801001\x00\x00\x00\x00',
```

Comment:
@ErichMoraga I know this has been a while, but do you know where you got this FW? I was sanity checking the FW versions and noticed this starts with `b'\x01'` but the version is 32 bytes (the first byte * 8 bytes is the length)

Reply:
Hey @sshane !!!  So glad someone finally asked me about that 2 years later.  I see July 28th, 2021 I said...
`@silentser#9447 2021 Highlander Hybrid (Germany) DongleID/route 4bc16a0e3b138d49|2021-07-28--11-10-55`
...and I actually still have a file on my laptop from that!

It shows that the aforementioned route included `carFw` as follows...
```
    carFw = [
      ( ecu = eps,
        fwVersion = "8965B48310\x00\x00\x00\x00\x00\x00",  OK
        address = 1953,
        subAddress = 0 ),
      ( ecu = esp,
        fwVersion = "\x01F152648C6300\x00\x00\x00\x00",
        address = 1968,
        subAddress = 0 ),
      ( ecu = engine,
        fwVersion = "\x01896630EA1000\x00\x00\x00\x00",
        address = 1792,
        subAddress = 0 ),
      ( ecu = fwdRadar,
        fwVersion = "\x018821F6201200\x00\x00\x00\x00",  OK
        address = 1872,
        subAddress = 15 ),
      ( ecu = fwdCamera,
        fwVersion = "\x028646F4803000\x00\x00\x00\x008646G5301200\x00\x00\x00\x00",
        address = 1872,
        subAddress = 109 ) ],
```